packages/node_modules/@ciscospark/react-component-activity-item-base/src/index.js

```diff
@@ -15,12 +15,14 @@ export default function ActivityItemBase(props) {
     children,
     id,
     isAdditional,
+    isError,
```

Comment:
Yea, I think `hasError` makes sense here because we are saying the item "is pending and has an error". I'll change that.